README.md

```diff
@@ -1,84 +1,61 @@
 
-```bash
-yarn add --dev jest-runner-tsd
+Note that since `v2` the `jest-runner-tsd` is using [`tsd-lite`](https://github.com/mrazauskas/tsd-lite) instead of [`tsd`](https://github.com/SamVerschueren/tsd). The type testing logic is the same in both implementations, i.e. both of them are wrapping around the [TypeScript compiler](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API) to analyze types of your code.
 
-# or with NPM
+## Differences from `tsd`
 
-npm install --save-dev jest-runner-tsd
-```
+The usage of `tsd` with monorepos written in TypeScript may become [cumbersome](https://github.com/SamVerschueren/tsd/issues/32), because of checks unrelated with types. `tsd-light` is an attempt to address these and similar issues.
 
-### Adding to Jest Config
+Notable differences:
 
-Create a `jest.config.types.js` file and have the runner property set to `jest-runner-tsd` as shown below:
+- The `tsd` configuration in `package.json` is ignored. Please use `jest.config` to configure discovery of your test files and `tsconfig.json` to provide configuration for TS compiler.
+- The compiler configuration will be read from the nearest `tsconfig.json` for each test file. Hence, you may have a configuration for the whole project, or a group of test files, or just a particular test file.
+- `tsd-lite` will use the default TS compiler configuration without altering it. This means you should set `"strict": true` to use strict assertions, which are the default ones in vanilla `tsd`.
+- Only type testing is performed without any additional checks or rules.
+- The `printType` helper is discarded.
 
-```js
-module.exports = {
-  runner: 'jest-runner-tsd',
-};
-```
-
-In the project `package.json` file, modify the scripts block to use the configuration file as show below:
-
-```json
-...
-"scripts": {
-  ...
-  "type-tests": "yarn jest --config jest.config.types.js"
-}
-...
-```
-
-### Run
-
-To start the test, just execute the following command
+## Install
 
 ```bash
-yarn test-types
+yarn add -D jest-runner-tsd @tsd/typescript
 ```
 
-## Writing tests
-
-> This runner uses TSD. To see the available assertions, checkout it's [documentation](https://github.com/SamVerschueren/tsd)
+Remember to install `@tsd/typescript` package. It is a required peer dependency.
 
-### For JavaScript Projects
+### Usage
 
-There are multiple ways you can pass a type definition file.
+1. If your type tests live inside `__typetests__` folders, set up `jest.config.tsd.js` like this:
 
-#### Default
-
-The type definitions should be in a file named `index.d.ts` in the root directory of the project by default.
+```js
+module.exports = {
+  displayName: {
+    color: 'blue',
+    name: 'types',
+  },
+  runner: 'jest-runner-tsd',
+  testMatch: ['**/__typetests__/**/*.test.ts'],
+};
+```
 
-#### `types` property in package.json
+2. Add [`tsconfig.json`](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) to your project with configuration for TS compiler.
```

Comment:
This is also unclear. An additional section on `Configuration` would do better job.